art/utils.py

```diff
@@ -434,7 +473,7 @@ def load_batch(fpath):
         data = data.reshape(data.shape[0], 3, 32, 32)
         return data, labels
 
-    from art import DATA_PATH
+    from art.utils import DATA_PATH
```

Comment:
Is this import still required? It results in a LGTM warning on importing itself.